src/components/GraphForRepo/useCommitList.ts

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { Commit, GitCommit } from "../../helpers/types";
+
+type ReturnType = {
+  commits: Commit[];
+  hasMore: boolean;
+  loadMore: () => Promise<void>;
+};
+
+export function useCommitList(
```

Comment:
leave data loading out of scope of the pakage, either leave it as example code in story book, or seperate out them as helper utilities.